corehq/apps/userreports/reports/util.py

```diff
@@ -0,0 +1,21 @@
+from django.utils.translation import ugettext
+
+
+def get_total_row(data, aggregation_columns, column_configs):
+    total_row = [0] * len(column_configs)
+    for i, column_config in enumerate(column_configs):
+        if column_config.calculate_total:
+            total_row[i] = reduce(
```

Comment:
sorry to nitpick, but i still feel like this function was designed to handle edge cases involving datatypes that aren't tested in the tests you added. possible to pull it out and exercise those?